src/components/LoadingModal.tsx

```diff
@@ -8,13 +7,12 @@ interface ILoadingModalProps {
 }
 
 const LoadingModal = ({ isVisible = true, children }: ILoadingModalProps) => {
-  const [isTxnProcessing] = useMCStore((s) => [s.isTxnProcessing]);
   return (
     <div
       className={cn(
-        'fixed left-0 top-0 z-[1000] flex h-full w-full items-center justify-center',
+        'fixed left-0 top-0 z-[900] flex h-full w-full items-center justify-center',
         {
-          hidden: !isVisible || !isTxnProcessing,
```

Comment:
I added this so when the user rejects the transaction, it will close the transaction modal

Reply:
I added a close modal trigger in `finally` block when creating a multisig account but haven't been able to test it because of multiclique install policy error